src/Consolonia.Core/Drawing/PixelBufferImplementation/BoxPattern.cs

```diff
@@ -1,61 +1,27 @@
 using System;
-using System.Diagnostics;
-using System.Diagnostics.CodeAnalysis;
-using System.Text.Json.Serialization;
-using Consolonia.Controls.Brushes;
 
 namespace Consolonia.Core.Drawing.PixelBufferImplementation
 {
     /// <summary>
     ///     https://en.wikipedia.org/wiki/Box-drawing_character
     /// </summary>
-    [DebuggerDisplay("DrawingBox {Text}")]
-    [JsonConverter(typeof(SymbolConverter))]
-    public readonly struct DrawingBoxSymbol : ISymbol, IEquatable<DrawingBoxSymbol>
+    public static class BoxPattern
     {
         // all 0bXXXX_0000 are special values
-        private const byte BoldSymbol = 0b0001_0000;
-        private const byte EmptySymbol = 0b0;
+        public const char Min = (char)0x2500;
+        public const char Max = (char)0x257F;
 
-        public DrawingBoxSymbol(byte upRightDownLeft)
-        {
-            UpRightDownLeft = upRightDownLeft;
-            Text = GetBoxSymbol(UpRightDownLeft).ToString();
-        }
-
-        public byte UpRightDownLeft { get; init; }
-
-        public bool Equals(DrawingBoxSymbol other)
-        {
-            return UpRightDownLeft == other!.UpRightDownLeft;
-        }
-
-        [JsonIgnore] public string Text { get; init; }
-
-        [JsonIgnore] public ushort Width => 1;
-
-        public bool NothingToDraw()
-        {
-            return UpRightDownLeft == EmptySymbol;
-        }
-
-        public ISymbol Blend(ref ISymbol symbolAbove)
-        {
-            if (symbolAbove.NothingToDraw()) return this;
-
-            if (symbolAbove is not DrawingBoxSymbol drawingBoxSymbol)
-                return symbolAbove;
-
-            if (drawingBoxSymbol.UpRightDownLeft == BoldSymbol || UpRightDownLeft == BoldSymbol)
-                return new DrawingBoxSymbol(BoldSymbol);
+        // all 0bXXXX_0000 are special values
+        public const char BoldChar = '█';
+        public const byte BoldMask = 0b0001_0000;
```

Comment:
But it's not mask, right? It's the value itself. Only one single possible value. Or is it mask? Because if it is just one value - we steal only one of 256 values. If it's mask - we still 32 values which all represent same.